core/embed/rust/src/ui/layout_eckhart/firmware/device_menu_screen.rs

```diff
@@ -136,11 +140,11 @@ pub struct DeviceMenuScreen<'a> {
     // as defined by `enum Subscreen` (DeviceScreen is still a VerticalMenuScreen!)
     // The active one will be Some(...) and the other one will be None.
     // This way we only need to keep one screen at any time in memory.
-    menu_screen: Option<VerticalMenuScreen>,
-    about_screen: Option<TextScreen<Paragraphs<[Paragraph<'a>; 2]>>>,
+    menu_screen: GcBox<Option<VerticalMenuScreen>>,
```

Comment:
Why not `Option<GcBox<...>>`?

Reply:
Right, looking at how `menu_screen` and `about_screen` are used it probably does make more sense.